worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -580,19 +580,23 @@ class FetchHandler(
         streamChunkSlice.chunkIndex,
         streamChunkSlice.offset,
         streamChunkSlice.len)
+      val bufSize = buf.size()
+      workerSource.updateHistogram(WorkerSource.FETCH_CHUNK_SIZE, bufSize)
       chunkStreamManager.chunkBeingSent(streamChunkSlice.streamId)
       client.getChannel.writeAndFlush(new ChunkFetchSuccess(streamChunkSlice, buf))
         .addListener(new GenericFutureListener[Future[_ >: Void]] {
           override def operationComplete(future: Future[_ >: Void]): Unit = {
             if (future.isSuccess) {
               if (log.isDebugEnabled) {
                 logDebug(
-                  s"Sending ChunkFetchSuccess to $remoteAddr succeeded, chunk $streamChunkSlice")
+                  s"Sending ChunkFetchSuccess to $remoteAddr succeeded," +
+                    s" chunk $streamChunkSlice, buf size: $bufSize")
               }
               workerSource.incCounter(WorkerSource.FETCH_CHUNK_SUCCESS_COUNT)
             } else {
               logWarning(
-                s"Sending ChunkFetchSuccess to $remoteAddr failed, chunk $streamChunkSlice",
+                s"Sending ChunkFetchSuccess to $remoteAddr failed," +
+                  s" chunk $streamChunkSlice, buf size: $bufSize",
```

Comment:
cc @FMX 
I just intended to log the real buf size to make the log clear, because the streamChunkSlice len is always Int.MaxValue.

Maybe the new added metrics is over design.